integrations-and-supported-tools/mlflow/scripts/run_examples.sh

```diff
@@ -0,0 +1,13 @@
+set -e
+
+echo "Installing requirements..."
+pip install -U -r requirements.txt
+
+echo "Running train_keras_mlflow.py..."
+python train_keras_mlflow.py
+
+echo "Running train_scikit_mlflow.py..."
+python train_scikit_mlflow.py
+
+echo "Export MLflow runs to Neptune..."
+neptune mlflow --project common/mlflow-integration
```

Comment:
How come `sync` is not needed here?

Reply:
This is just the exporter to export historical MLflow runs to Neptune. The `sync` feature is still under development

Reply:
Also, `sync` would most probably be a python client option, not CLI

Reply:
I'm quite confused. I thought it was part of the exporter command: https://github.com/neptune-ai/neptune-mlflow/blob/6d889e1b2a875e443c7b28f170b01fd8ab85492e/src/neptune_mlflow_exporter/__init__.py#L41C4-L41C4

Reply:
Nope, thats the method called by `neptune mlflow` command. The `click.options` are additional flags you can add.

```bash
❯ neptune mlflow --help
Usage: neptune mlflow [OPTIONS]

  Exports MLflow runs to neptune.ai.

  Args:     project: name of the Neptune project where your MLflow runs will
  go.         If not provided, NEPTUNE_PROJECT env variable will be used.
  api_token: your Neptune API token.         If not provided,
  NEPTUNE_API_TOKEN env variable will be used.     mlflow_tracking_uri: your
  MLflow tracking URI.         If not provided, it is left to the MLflow
  client to resolve it.     exclude_artifacts: whether to skip uploading
  artifacts to Neptune.     max_artifact_size: max size of the artifact to be
  uploaded to Neptune.         Unit is in MB.         For directories this
  will be treated as the max size of the entire directory.

Options:
  -p, --project TEXT              Neptune project name
  -a, --api-token TEXT            Your Neptune API Key
  -u, --mlflow-tracking-uri TEXT  Your MLflow tracking URI
  -e, --exclude-artifacts BOOLEAN
                                  Specifies whether to also include artifacts
                                  in the upload
  -m, --max-artifact-size INTEGER
                                  Maximum size uploaded to Neptune, unit is in
                                  MB
  --help                          Show this message and exit.
  ```

Reply:
Ahh, got it. Thanks for explaining!